includes/definitions/siteboss.yaml

```diff
@@ -0,0 +1,12 @@
+os: siteboss
+text: 'Asentria SiteBoss'
+type: server
+icon: siteboss
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+mib_dir: asentria
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.3052.12.3.2
```

Comment:
Is this worth changing to `.1.3.6.1.4.1.3052.12.3.` or `.1.3.6.1.4.1.3052.12.`. Our stats show no other devices on either of these.

Reply:
I'll look through some MIBs and see if I can find any obvious reason not to.